README.md

```diff
@@ -72,27 +72,8 @@ save --mode=single --language=java
 ```
 
 # <a name="config_options"></a> List of options Save cli
```

Comment:
I think instead of this header we can just write `For the complete list of supported options refer to the [options table](/OptionsTable.md)`

Reply:
Done